research-plan.md

```diff
@@ -0,0 +1,75 @@
+# FOIA platform research plan
+
+This is a living document. It will be updated over the course of the coming weeks as we learn more from our users and as we hone in our hypotheses.
+
+## Background
+
+Initial research on National FOIA Portal. Attempting to focus on speaking to user groups who were not heavily represented in previous FOIA 1.0 research.
+
+The goal of this research is to inform hypotheses that address the following:
+- How might we create a platform that improves the FOIA requesting experience?
+- How might we create a platform that increases the quality of FOIA requests?
+- How might various FOIA requesters use a national FOIA platform?
```

Comment:
I might include the agency perspective. Based on what we've heard and learned so far, I think this is somewhere we can make a dent, even indirectly.

> How might agency processes be improved or enhanced by a national FOIA platform?